shotwell/README.md

```diff
@@ -0,0 +1,18 @@
+# Shotwell snap
+
+This project creates a working snap of Shotwell from gnome git.
+
+To get this done, we need to do the following:
+ - build from current git
+
+## Current state
+
+Working features:
+ - Photo import from folder
+
+Known issues:
+  - Only starts if installed with --devmode
+
+TODO:
+ -
+ 
```

Comment:
Maybe add that it should be worked out why this doesn't work without `--devmode`? Can you maybe bring it up as a question on snapcraft@lists.snapcraft.io?